fireant/tests/slicer/widgets/test_reacttable.py

```diff
@@ -2,7 +2,7 @@
 from unittest import TestCase
 
 from fireant.slicer.totals import MAX_STRING
-from fireant.slicer.widgets.reacttable import ReactTable
+from fireant.slicer.widgets.reacttable import ReactTable, ReferenceItem
```

Comment:
can you please fix your pycharm formatted to enable multiple imports to be chopped down onto multiple lines? this lowers the risk of running into merge conflicts.